gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapper.java

```diff
@@ -38,6 +39,30 @@ public void mergeDuplicateObservationStatements(RCMRMT030101UKEhrExtract ehrExtr
                 .forEach(DuplicateObservationStatementMapper::mergeDuplicateObservationStatements);
     }
 
+    private static @NotNull Stream<RCMRMT030101UKEhrComposition> getEhrCompositionFromEhrExtract(RCMRMT030101UKEhrExtract ehrExtract) {
+        return ehrExtract.getComponent()
+            .stream()
+            .map(RCMRMT030101UKComponent::getEhrFolder)
+            .map(RCMRMT030101UKEhrFolder::getComponent)
+            .flatMap(List::stream)
+            .map(RCMRMT030101UKComponent3::getEhrComposition);
+    }
+
+    public void removeDuplicateObservationStatements(RCMRMT030101UKEhrExtract ehrExtract) {
```

Comment:
Couldn't this behaviour just be merged into the existing `mergeDuplicateObservationStatements` ?

Reply:
sorry, there is still work in progress on this ticket. I just mistakenly raised a normal PR instead of a draft one